variantlib/variant_file.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import logging
+
+from variantlib.constants import VARIANTS_JSON_VARIANT_DATA_KEY
+from variantlib.errors import ValidationError
+from variantlib.models.variant import VariantDescription
+from variantlib.validators import validate_variant_json
+
+logger = logging.getLogger(__name__)
+
+
+def unpack_variants_json(
+    variants_json: dict[str, dict[str, dict]],
```

Comment:
```suggestion
    variants_json: dict,
```

`variants.json` can have other top-level keys whose values may not be `dict`s.

Reply:
Ah right